app/initstores.go

```diff
@@ -135,6 +136,13 @@ func (app *App) initStores(ctx context.Context) error {
 		return errors.Wrap(err, "init service store")
 	}
 
+	if app.BasicStore == nil {
+		app.BasicStore, err = basic.NewStore(ctx, app.db)
```

Comment:
Update `basic.NewProvider` to take the `Store` as an arg instead of `*sql.DB` and pass it this, so we don't initialize two.